content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -159,6 +157,10 @@ link:http://groovy-lang.org/objectorientation.html#_annotation[annotation]
 pattern `@Library('my-shared-library') _` may be useful for keeping code
 concise. In essence, instead of annotating an unnecessary `import` statement,
 the symbol `_` is annotated.
+
+It is *not* recommended to `import` a global variable/function,
+since this will force the compiler to interpret fields and methods as `static`
+even if they were intended to be instance.
```

Comment:
Try to avoid. You just get a confusing error message from Groovy if you had declared an instance member. Something like “No such property: foo; perhaps you meant: foo?”